foodnet/membership/models.py

```diff
@@ -94,6 +106,13 @@ class Department(models.Model):
     def __str__(self):
         return u'{0}'.format(self.name)
 
+    @property
+    def accounts(self):
+        out = []
+        for member in self.members.all():
+            out += member.userprofile_set.all()
```

Comment:
Am I right in thinking that when a user belongs to more than one membership let say a school and his household, this will have a duplicate? If so, calling `set` won't help either but we could simply add manager, couldn't we?